packages/amplify_datastore/ios/Classes/SwiftAmplifyDataStorePlugin.swift

```diff
@@ -110,20 +114,32 @@ public class SwiftAmplifyDataStorePlugin: NSObject, FlutterPlugin {
         let syncPageSize = args["syncPageSize"] as? UInt ?? DataStoreConfiguration.defaultSyncPageSize
         
         do {
-            
-            let modelSchemas: [ModelSchema] = try modelSchemaList.map {
-                try FlutterModelSchema.init(serializedData: $0).convertToNativeModelSchema()
+            let customTypeSchemaDeserializingOrder = try getCustomTypeSchemasDependentOrder(
```

Comment:
*nit:* should this be `customTypeSchemaDependentOrder` or `customTypeSchemaDependenciesOrder`? Some of the naming is a little consistent. The var here uses "DeserializingOrder", the function uses "DependentOrder", and the method `registerCustomTypeSchemas` uses "dependenciesOrder".